charts/renku/requirements.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
   alias: graph
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
   version: 0.1.0-aa0b94b
```

Comment:
Does it make sense to include the graph at all before there has been a first graph release?

Reply:
Yes because we're using it for development. Otherwise I have to make a release branch just for this.

Reply:
I agree, that would be a bit annoying.

Reply:
Could we then go with version `0.1.0-36eedf7`? `0.1.0-aa0b94b` is pretty old and differs not just in functionality but also in the way dataset is and postgres db are created.